2006-11-09  Ulrich Drepper  <drepper@redhat.com>

	[BZ #3493]
	* posix/unistd.h (sysconf): Remove const attribute.

--- libc/posix/unistd.h	24 Aug 2006 06:46:27 -0000	1.147
+++ libc/posix/unistd.h	9 Nov 2006 21:51:58 -0000	1.148
@@ -559,7 +559,7 @@ extern long int pathconf (__const char *
 extern long int fpathconf (int __fd, int __name) __THROW;
 
 /* Get the value of the system variable NAME.  */
-extern long int sysconf (int __name) __THROW __attribute__ ((__const__));
+extern long int sysconf (int __name) __THROW;
 
 #ifdef	__USE_POSIX2
 /* Get the value of the string-valued system variable NAME.  */
